apps/client-server/src/app/file/file.service.spec.ts

```diff
@@ -76,6 +77,7 @@ describe('FileService', () => {
         CreateFileService,
         UpdateFileService,
         FileService,
+        ValidationService,
```

Comment:
I've tried to fix a lot of tests. The problem was only with their dependencies, tho im not really experienced in nest to do so some of the tests are still failling

Reply:
Yeah sorry they are somewhat messy right now, I will work to fix them up.
Usually just some dep injection being missed somewhere. Should probably move a lot of these to use mocks instead..